source/Nuke.Common/Execution/Telemetry.cs

```diff
@@ -61,7 +61,10 @@ string GetCookieFile(string name, int version)
                 var cookieFile = GetCookieFile(cookieName, CurrentVersion);
                 if (!File.Exists(cookieFile))
                 {
-                    PrintDisclosure($"create awareness cookie for {cookieName.SingleQuote()}");
+                    if (NukeBuild.IsServerBuild)
```

Comment:
Feels like `NukeBuild.IsServerBuild` isn't the right abstraction here - seems like we should have a `NukeBuild.IsInteractive` property instead?